data/transport missions.txt

```diff
@@ -1868,6 +1872,309 @@ mission "Terraforming 5"
+			`	The representative falls silent. After a moment of thinking, he says "Captain, I need you to take me to Parliament. I may be able to authorize the use of a device that should suffice, but it may not be as cheap as Ms. Amy is hoping."`
+			choice
+				`	"I'll get my ship ready to launch."`
+				`	"Sorry, I have other things I could be doing."`
+					decline
+			
+			`	The representative leaves the bar. "We'll wait here," Amy says to you. "I'll speak with the Tundra government about evacuating the area around where the impact site will be so that we can start when you get back."`
+				accept
+
+
+
+mission "Terraforming 11"
+	landing
+	name "Terraforming Tundra"
+	description "Return to <destination> with the Republic representative."
+	destination "Tundra"
+	passengers 1
+	to offer
+		has "Terraforming 10: done"
+	
+	on offer
+		conversation
+			`"Wait here," the Republic representative says in a curt tone. Several hours later, he returns, not looking very happy. "What a waste of time. Parliament has declined my request. I don't much agree with this, but you will need to crash an asteroid into Tundra if you wish to achieve your goal." Before you ready for launch, you send a message to Amy and Nolan telling them that an asteroid will be used after all. You don't receive an immediate response.`
+				accept
+	
+	on enter "Cebalrai"
+		dialog
+			`Your ship is targeted by pirate fleets when you enter the system. Before you engage, you receive a message from Amy.`
+			`"Captain, these pirate ships have been occupying the system for days. You must take them out if we are to safely crash an asteroid into Tundra."`
+	npc evade
+		government "Bounty Hunter"
+		personality nemesis staying target
+		system "Cebalrai"
+		fleet "Marauder fleet I"
+		fleet "Large Southern Pirates"
+
+
+
+mission "Terraforming 12"
+	landing
+	name "Terraforming Tundra"
+	description "Plant a thruster on an asteroid to guide it into a collision with the dormant supervolcano on Tundra."
+	cargo "thruster equipment" 10
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Terraforming 11: done"
+	
+	on offer
+		payment 300000
+		conversation
+			`The pirates that occupied the system did not make an effort to land on Tundra, but they did harass any merchant ships that passed through the system. The government of Tundra pays you <payment> for ridding the system of the pirates, a pleasantly unexpected reward.`
+			`	You gather with the scientists in the spaceport bar. The Tundra native informs you that the area around the impact zone has been evacuated. "All we need to do now is find the right asteroid" After only a short time of searching, Amy finds the perfect candidate.`
```

Comment:
Supervolcano means evacuating more than just the impact area.... at least a quarter of the continental US is screwed by the Yellowstone caldera, and I can't imagine this would be different.